metabase/metabase
# Add new parameter source modal

Epic https://github.com/metabase/metabase/issues/26898

Redesigns the modal to configure the parameter source.

Known issues (to be fixed separately):
- Currently `Dropdown list` is the only option available in parameter sidebar settings.
- Multiple cards can be selected while only the first one is used.
- No preview of selected card data.

How to verify:
- Open a dashboard with a card
- Edit -> Filter -> Text -> Dropdown
- Connect the parameter to the card
- Click `Dropdown list` -> `Edit` in the parameter sidebar
- Click `Custom list` and enter custom values -> `Done`
- Save the dashboard
- It should be possible to select custom values from the dropdown for the parameter now


<img width="389" alt="Screenshot 2023-01-09 at 13 43 31" src="https://user-images.githubusercontent.com/8542534/211302653-07c232c7-361a-457c-b268-d8c3364ca5fb.png">
<img width="1137" alt="Screenshot 2023-01-09 at 16 24 18" src="https://user-images.githubusercontent.com/8542534/211331154-838d4ac3-74a5-4402-82c7-a105b982c066.png">
<img width="1142" alt="Screenshot 2023-01-09 at 16 24 30" src="https://user-images.githubusercontent.com/8542534/211331186-5cdd6107-b98f-4b5d-9703-1605286cf5aa.png">
<img width="1142" alt="Screenshot 2023-01-09 at 17 27 07" src="https://user-images.githubusercontent.com/8542534/211344587-a9df8ae9-8503-4323-8700-530ec2c3a919.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27530)
